src/test/java/life/calgo/testutil/TypicalDailyFoodLog.java

```diff
@@ -22,6 +22,11 @@ public static DailyFoodLog getAppleOnlyLog() {
         return APPLE_ONLY_FOOD_LOG;
     }
 
+    public static DailyFoodLog getBananaOnlyLog() {
+        DailyFoodLog BANANA_ONLY_FOOD_LOG = new DailyFoodLog().consume(TypicalFoodItems.BANANA ,1);
```

Comment:
@J-Dan23 sorry sis, I used the wrong naming convention previously. Do take note to change this to bananaOnlyFoodLog.

Reply:
Noted!